src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -101,17 +101,24 @@
     public static final String FILTER_NO_TARGET = "NoTarget";
 
     /**
-     * Filter name to use when a target is lifted over, but the reference allele doens't match the new reference.
+     * Filter name to use when a target is lifted over, but the reference allele doesn't match the new reference.
      */
     public static final String FILTER_MISMATCHING_REF_ALLELE = "MismatchedRefAllele";
 
     /**
+     * Filter name to use when an indel cannot be lifted over since it straddles two links which means
+     * that it is unclear what are the right alleles to be used.
+     */
+    public static final String FILTER_INDEL_STRADDLES_TWO_LINKS = "IndelStraddlesMultipleLinks";
```

Comment:
My only major issue with this PR is that I haven't got a clue what this means.  What on earth is an indel that straddles multiple links?!  I'm thinking that it means that it overlaps multiple intervals from the chain file?  If so, I think that "links" is the wrong term here; better just to use intervals.  So I would say "IndelStraddlesMultipleChainIntervals"